script/setup.js

```diff
@@ -36,6 +36,8 @@ try {
 		strict: false,
 	});
 
+	values["skip-api"] = true;
```

Comment:
Aha! This is a nifty way to solve the issue 😄 nice one.

At first I was -1 on this because it means `--skip-api false` isn't supported... but it looks like that's already the case!

Another issue though, `Pre-filling skip-api to true.` gets logged to the console even if someone provides `--skip-api` in the terminal. Not ideal.

Suggestion: I think it's fine to not use `getPrefillOrPromptedValue`. Directly calling to `prompt` or similar is probably fine. What do you think?

Reply:
Oh yeah, sorry for the late response.
If removing the console log is not a big deal, maybe we can go that route :thinking:

Reply:
@TAKANOME-DEV just checking in, is this still something you have time for?

Reply:
@TAKANOME-DEV I just pushed this fix now so I can share this package with some folks - cheers!